8dev_restapi.js

```diff
@@ -552,7 +552,13 @@ class Service extends EventEmitter {
     return new Promise((fulfill, reject) => {
       this.get('/notification/callback').then((dataAndResponse) => {
         if (dataAndResponse.resp.statusCode === 200) {
-          fulfill(dataAndResponse.data);
+          if (dataAndResponse.data.url === `http://${this.ipAddress}:${this.config.port}/notification`) {
+            fulfill(dataAndResponse.data);
+          } else {
+            const err = new Error('Notification callback IP adress does not match service configuration');
```

Comment:
Klaidos priežastį galima generalizuot (nes ne vien IP gali būt blogas) ir sutrumpint: 'Notifcation callback does not match current configuration'.
Taip pat jeigu jau naudojamas error.code, geriau būtų laikytis rekomendacijos (https://nodejs.org/api/errors.html#errors_error_code_1), pvz 'EINVALIDCALLBACK'.